backend/coins/eth/etherscan/etherscan.go

```diff
@@ -39,7 +39,7 @@ import (
 
 // CallInterval is the duration between etherscan requests.
 // Etherscan rate limits to one request per 0.2 seconds.
-var CallInterval = 210 * time.Millisecond
+var CallInterval = 260 * time.Millisecond
```

Comment:
I'd make it a nice round number of 300 then :)

Reply:
Step by step ;)